jsonpb.go

```diff
@@ -0,0 +1,20 @@
+package jsonpb
+
+import (
+	"bytes"
+
+	"github.com/golang/protobuf/jsonpb"
```

Comment:
Take it or leave it: For clarity I like to alias packages that come in and share a name with the current package or any global level declarations within the current package.